src/network/networkd-network-gperf.gperf

```diff
@@ -110,6 +110,9 @@ Bridge.AllowPortToBeRoot,               config_parse_bool,
 Bridge.UnicastFlood,                    config_parse_bool,                              0,                             offsetof(Network, unicast_flood)
 BridgeFDB.MACAddress,                   config_parse_fdb_hwaddr,                        0,                             0
 BridgeFDB.VLANId,                       config_parse_fdb_vlan_id,                       0,                             0
+BridgeVLAN.PVID,                        config_parse_uint32,                            0,                             offsetof(Network, pvid)
```

Comment:
also, I think there should be config_parse_vlanid() or so that is used here and checks the range to be valid.